tests/test_edgeql_insert.py

```diff
@@ -1715,3 +1715,121 @@ async def test_edgeql_insert_correlated_bad_02(self):
                     Subordinate,
                 );
             ''')
+
+
+class TestInsertCoalesce(tb.QueryTestCase):
```

Comment:
I wouldn't add another class as that would slow down test start time (every test case gets its own DB which takes some time).